### DISCUSSION TITLE
群晖7.2 docker里运行wrt，只有lo 接口启动。

### DISCUSSION THREAD

**Discussion by Mrkang168** [immortalwrt/immortalwrt#1415] [Q&A]
### Describe the bug

群晖 网络环境 配置。
日志提示:sh: write error: I/O error，容器起来以后看着是获取到了macvlan的IP地址
运行环境是黑裙DSM7.2.1，打开了open vswitch和promisc on都不行，感觉上跟运行环境有关系

1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1
link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
inet 127.0.0.1/8 scope host lo
valid_lft forever preferred_lft forever
inet6 ::1/128 scope host
valid_lft forever preferred_lft forever
2: sit0@NONE: mtu 1480 qdisc noop state DOWN group default qlen 1
link/sit 0.0.0.0 brd 0.0.0.0
3: eth1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
link/ether 00:11:32:e1:e9:04 brd ff:ff:ff:ff:ff:ff
inet 192.168.30.85/24 brd 192.168.30.255 scope global eth1
valid_lft forever preferred_lft forever
4: eth0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc pfifo_fast state DOWN group default qlen 1000
link/ether 00:11:32:e1:e9:03 brd ff:ff:ff:ff:ff:ff
inet 169.254.26.84/16 brd 169.254.255.255 scope global eth0
valid_lft forever preferred_lft forever
5: docker0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
link/ether 02:42:2a:37:32:30 brd ff:ff:ff:ff:ff:ff
inet 172.17.0.1/16 brd 172.17.255.255 scope global docker0
valid_lft forever preferred_lft forever
inet6 fe80::42:2aff:fe37:3230/64 scope link
valid_lft forever preferred_lft forever
7: docker8a028dd@if6: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master docker0 state UP group default
link/ether 0a:4a:da:07:46:38 brd ff:ff:ff:ff:ff:ff link-netnsid 0
inet6 fe80::84a:daff:fe07:4638/64 scope link
valid_lft forever preferred_lft forever
20: docker55c4183@if19: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master docker0 state UP group default
link/ether 86:4a:13:65:4e:7a brd ff:ff:ff:ff:ff:ff link-netnsid 1
inet6 fe80::844a:13ff:fe65:4e7a/64 scope link
valid_lft forever preferred_lft forever

命令运行：docker run -d --restart always --name zzopenwrt --network macnet --privileged registry.cn-hangzhou.aliyuncs.com/zzsrv/openwrt:x86_64 /sbin/init

network config 如下：
cat /etc/config/network

config interface 'loopback'
option proto 'static'
option ipaddr '127.0.0.1'
option netmask '255.0.0.0'
option device 'lo'

config globals 'globals'
option packet_steering '1'

config interface 'lan'
option proto 'static'
option netmask '255.255.255.0'
option ip6assign '60'
option ipaddr '192.168.1.2'
option gateway '192.168.1.1'
option dns '192.168.1.1'
option device 'br-lan'

config device
option name 'br-lan'
option type 'bridge'
list ports 'eth0'

config interface 'utun'
option proto 'none'
option ifname 'utun'
option device 'utun'

但是使用Ifconfig查看：
ifconfig
lo Link encap:Local Loopback
inet addr:127.0.0.1 Mask:255.0.0.0
inet6 addr: ::1/128 Scope:Host
UP LOOPBACK RUNNING MTU:65536 Metric:1
RX packets:0 errors:0 dropped:0 overruns:0 frame:0
TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
collisions:0 txqueuelen:1
RX bytes:0 (0.0 B) TX bytes:0 (0.0 B)

只有这个一个接口。

请问是哪里的问题？

### ImmortalWrt version

23.05

### ImmortalWrt release

23.05

### ImmortalWrt target/subtarget

23.05

### Device

x86 64

### Image kind

Self-built image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by 1715173329** [comment]
使用第三方打包的固件，请联系第三方处理。
没有内核日志，无法诊断问题。
